sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -935,8 +935,9 @@ class AdaptiveQueryExecSuite
           val error = intercept[SparkException] {
             joined.collect()
           }
-          assert((Seq(error) ++ Option(error.getCause) ++ error.getSuppressed()).exists(
-            e => e.getMessage() != null && e.getMessage().contains("coalesce test error")))
+          val exceptionMessages = (Seq(error) ++ Option(error.getCause) ++ error.getSuppressed())
```

Comment:
Is this change necessary? The issue "SPARK-47148: AQE should avoid submitting a shuffle job upon cancellation" is associated with a flaky test.

Reply:
I can undo this. The test failed during one run for me and I made this change to try to debug it.
This change makes it easier for Scalatest to infer a test failure message because the collection being asserted on is a `Seq[String]` instead of `Seq[Exception]`.